packages/ember-glimmer/lib/environment.js

```diff
@@ -275,16 +275,16 @@ export default class Environment extends GlimmerEnvironment {
     this.inTransaction = true;
 
     super.begin();
-
-    this.destroyedComponents = [];
   }
 
   commit() {
+    let destroyedComponents = this.destroyedComponents;
+    this.destroyedComponents = [];
```

Comment:
Why do this assignment here. Why not just reassign to an empty array after the subsequent loop? More of a stylistic thing I think.